### ISSUE TITLE
Props files and publicizer.

### ISSUE THREAD

**Issue by Sl4vP0weR** [krafs/Publicizer#62]
I have some issues with inserting publicizer in my props file.
Here's my [line with usage of publicizer](https://github.com/SlavRim/ExampleMod/blob/ed3d3478571032f926c85a42ff51a04c300632ce/Shared.mod.props.csproj#L69) in my props file. 
First I was getting errors trying to migrate from version 1.0.2 to 2.2.0.
Looks like my IDE and .NET CLI restore doesn't updated the version, but targets of new versions was used.
My guess is that props file somehow not supported.
I've fixed it by moving lines related to publicizer into csproj, but is there some other solution to use it in props?

**Comment by krafs** [comment]
There shouldn't be any issues with having the config in props-files. I'll see if I can test it on my end.

**Comment by krafs** [comment]
So, I cloned your project and built it.
I noticed I had to specify the configuration if building from the command line, or ColoredText wouldn't get referenced properly.
`dotnet build .\ExampleMod.csproj -c 1.4_DEBUG`

But it seems to work fine for me. I also tried updating to latest Publicizer 2.2.0. Also seems to work, as long as you close VS, clear the cache (.vs and obj), and restart VS.

Am I testing the right thing, or was there something in particular that doesn't work?
